mlir/lib/Dialect/MIOpen/Transforms/BlockwiseGemmToThreadwise.cpp

```diff
@@ -240,10 +241,17 @@ struct BlockwiseGemmV2RewritePattern
                                 ConversionPatternRewriter &b) const override {
     Location loc = op.getLoc();
 
+    int64_t M = op->getAttr("m").template cast<IntegerAttr>().getInt();
```

Comment:
Is this a temporary measure until we've done some stuff with transforms, or should we go ahead and put `m`, `n`, and `k` as attributes in the operation definition?

Reply:
I mean that, instead of implicitly assuming they're there, they should be listed as required attributes in `MIOpenOps.td` ... unless we're getting rid of them through coordinate transforms stuff and so can get `M`, `N`, `K`, and `KPack` from the argument types ... which could be either this PR or a future PR

Reply:
>  they should be listed as required attributes in MIOpenOps.td ... which could be either this PR or a future PR

Thanks for the reminder and I can confirm with you that this is on my list. The goal is to catch up with nonxdlops side of things you have done.